pkg/scannerv4/mappers/mappers.go

```diff
@@ -369,6 +368,15 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 				description = nvdVuln.Descriptions[0].Value
 			}
 		}
+		issued := issuedTime(v.Issued, nvdVuln.Published)
+		if issued == nil {
+			zlog.Warn(ctx).
+				Err(err).
```

Comment:
`err` here would likely be always nil (or refer to something else), perhaps remove?

Is it worth logging the values of `v.Issued` and/or `nvdVuln.Published` so that we can determine if the value was just 'missing' or some format we don't recognize?